doc/particles.md

```diff
@@ -5,85 +5,88 @@ Flame offers a basic, yet robust and extendable particle system. The core concep
 The most basic usage of `Particle` with `BaseGame` would look as following:
 
 ```dart
-import 'package:flame/components/particle_component.dart';
+import 'package:flame/components.dart';
 
 // ... 
 
 game.add(
-    // Wrapping a [Particle] with [ParticleComponent]
-    // which maps [Component] lifecycle hooks to [Particle] ones
-    // and embeds a trigger for removing the component.
+    /// Wrapping a [Particle] with [ParticleComponent]
+    /// which maps [Component] lifecycle hooks to [Particle] ones
+    /// and embeds a trigger for removing the component.
     ParticleComponent(
         particle: CircleParticle()
     )
 );
 ```
 
-When using `Particle` with custom `Game` implementation, please ensure that `Particle` `update` and `render` lifecycle hooks are called during each game loop frame.
+When using `Particle` with a custom `Game` implementation, please ensure that both the `update` and `render` methods are called during each game loop tick.
 
 Main approaches to implement desired particle effects:
-* Composition of existing behaviors
-* Use behavior chaining (just a syntactic sugar of the first one)
-* Using `ComputedParticle`
+* Composition of existing behaviors.
+* Use behavior chaining (just a syntactic sugar of the first one).
+* Using `ComputedParticle`.
 
-Composition works in a similar fashion to those of Flutter widgets by defining the effect from top to bottom. Chaining allows to express same composition trees more fluently by defining behaviors from bottom to top. Computed particles in their turn fully delegate implementation of the behavior to your code. Any of the approaches could be used in conjunction with existing behaviors where needed.
+Composition works in a similar fashion to those of Flutter widgets by defining the effect from top to bottom. Chaining allows to express the same composition trees more fluently by defining behaviors from bottom to top. Computed particles in their turn fully delegate implementation of the behavior to your code. Any of the approaches could be used in conjunction with existing behaviors where needed.
 
 Below you can find an example of a effect showing a burst of circles, accelerating from `(0, 0)` to a random directions using all three approaches defined above.
 ```dart
 Random rnd = Random();
-Function randomOffset = () => Offset(
+
+Vector2 randomVector2() => Vector2(
     rnd.nextDouble() * 200 - 100, 
     rnd.nextDouble() * 200 - 100,
 );
 
-// Composition
-// Defining particle effect as set of nested
-// behaviors from top to bottom, one within another: 
-// ParticleComponent 
-//   > ComposedParticle 
-//     > AcceleratedParticle 
-//       > CircleParticle
+/// Composition.
+/// 
+/// Defining particle effect as set of nested behaviors from top to bottom, one within another: 
```

Comment:
```suggestion
/// Defining a particle effect as a set of nested behaviors from top to bottom, one within another: 
```

Reply:
Hmm, this sentence is a bit weird overall.

Reply:
same here!

Reply:
Any good suggestions for rewriting?